docs/en/observability/whats-new.asciidoc

```diff
@@ -3,24 +3,94 @@
 
 Here are the highlights of what's new and improved in {minor-version}.
 
-// Other versions:
-// {observability-guide-all}/8.12/whats-new.html[8.12] |
-// {observability-guide-all}/8.11/whats-new.html[8.11] |
-// {observability-guide-all}/8.10/whats-new.html[8.10] |
-// {observability-guide-all}/8.9/whats-new.html[8.9] |
-// {observability-guide-all}/8.8/whats-new.html[8.8] |
-// {observability-guide-all}/8.7/whats-new.html[8.7] |
-// {observability-guide-all}/8.6/whats-new.html[8.6] |
-// {observability-guide-all}/8.5/whats-new.html[8.5] |
-// {observability-guide-all}/8.4/whats-new.html[8.4] |
-// {observability-guide-all}/8.3/whats-new.html[8.3] |
-// {observability-guide-all}/8.2/whats-new.html[8.2] |
-// {observability-guide-all}/8.1/whats-new.html[8.1] |
-// {observability-guide-all}/8.0/whats-new.html[8.0] |
-// {observability-guide-all}/7.17/whats-new.html[7.17]
+Other versions:
+{observability-guide-all}/8.12/whats-new.html[8.12] |
+{observability-guide-all}/8.11/whats-new.html[8.11] |
+{observability-guide-all}/8.10/whats-new.html[8.10] |
+{observability-guide-all}/8.9/whats-new.html[8.9] |
+{observability-guide-all}/8.8/whats-new.html[8.8] |
+{observability-guide-all}/8.7/whats-new.html[8.7] |
+{observability-guide-all}/8.6/whats-new.html[8.6] |
+{observability-guide-all}/8.5/whats-new.html[8.5] |
+{observability-guide-all}/8.4/whats-new.html[8.4] |
+{observability-guide-all}/8.3/whats-new.html[8.3] |
+{observability-guide-all}/8.2/whats-new.html[8.2] |
+{observability-guide-all}/8.1/whats-new.html[8.1] |
+{observability-guide-all}/8.0/whats-new.html[8.0] |
+{observability-guide-all}/7.17/whats-new.html[7.17]
 
 // tag::whats-new[]
 
-// What's new content goes in here. Don't uncomment or remove the tags surrounding this content :)
+Elastic Observability 8.13 announces general availability (GA) for AWS Bedrock support in Elastic AI Assistant, along with enhancements for the AI Assistant and SLOs.
+
+[float]
+[[ai-assistant-enhancements]]
+== AI Assistant enhancements
+
+* Elastic AI Assistant support for AWS Bedrock
+* Contextual awareness of the entry point into AI Assistant
+* Performance improvements
+* Tech preview of editing and visualizing ES|QL queries from the AI Assistant
+
+[float]
+[[aws-bedrock-support]]
+=== AWS Bedrock support
+
+Elastic Observability 8.13 adds connectivity for AWS Bedrock, specifically for the Claude models. The Bedrock/Claude LLM adapter parses data from AWS using a SerDe stream using `@smithy/eventstream-serde-nodeuses` and offers the same experience as the other Elastic LLM connectors (OpenAI and Azure OpenAI). Setup is simple with the Amazon Bedrock connector configuration page.
```

Comment:
Not sure about this, but I see `@smithy/eventstream-serde-node` online, but not `nodeuses`. However I don't really know anything about this particular instance, just looked off.

Reply:
```suggestion
Elastic Observability 8.13 adds connectivity for AWS Bedrock, specifically for the Claude models. The Bedrock/Claude LLM adapter parses data from AWS using a SerDe stream using `@smithy/eventstream-serde-node` and offers the same experience as the other Elastic LLM connectors (OpenAI and Azure OpenAI). Setup is simple with the Amazon Bedrock connector configuration page.
```